.github/workflows/asv_check.yml

```diff
@@ -0,0 +1,39 @@
+name: CI ASV CHECK
+
+# CI ASV CHECK is aimed to verify that the benchmarks execute without error.
+on: [pull_request, push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    defaults: 
+      run:
+        shell: bash -el {0}
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Install Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.9.7'
+
+      - name: Install asv
+        run: |
+          pip install .
+          pip install ephem==3.7.6.0
+          pip install numba==0.40.0
+          pip install asv==0.4.2
+
+      - name: Run asv benchmarks
+        run: |
+          cd benchmarks
+          asv machine --yes
+          asv run --quick --dry-run --show-stderr --python=same\
+          | sed "/failed$/ s/^/##[error]/" | tee benchmarks.log
```

Comment:
I'm not sure the backslash works as a line continuation here.  I'd just put it all on one line instead of trying to figure out whatever weird YAML syntax is needed to make a multi-line command work.